pype/plugins/ftrack/publish/collect_ftrack_family.py

```diff
@@ -0,0 +1,41 @@
+"""
+Requires:
+    none
+
+Provides:
+    instance     -> families ([])
+"""
+import re
+import pyblish.api
+
+
+class CollectFtrackFamily(pyblish.api.InstancePlugin):
+    """
+        Adds explicitly 'ftrack' to families to upload instance to FTrack.
+
+        Uses selection by combination of hosts/families/tasks names (regex)
+    """
+    label = "Collect FTrack Family"
+    order = pyblish.api.CollectorOrder + 0.4999
+
+    hosts = ["standalonepublisher"]
+    families = ["render", "image"]
```

Comment:
I would not do family filtering on plugin level but rather let do it using profiles (like in extract review) so it can be easily ported to other hosts just modifying `hosts` attribute in future and targeting to add ftrack family preciselly.
```
{
    "hosts": [],
    "families": [],
    "task_names": [],
    "add_ftrack_family": true
}
```

Reply:
Still keep `hosts` attribute there. `hosts = ["standalonepublisher"]` is important for plugin filtering (has nothing to do with profiles).